docusaurus/docs/reactnative/basics/offline-support.mdx

```diff
@@ -26,12 +26,12 @@ The following features are currently **NOT** implemented. They will be impleme
 
 First and foremost, make sure to follow all steps from [Migrating to v5](../basics/migrating-from-4.x-to-5.x.mdx) guide. To enable offline support, please follow the given steps:
```

Comment:
```suggestion
First and foremost, make sure to follow all steps from [Migrating to v5](../basics/migrating-from-4.x-to-5.x.mdx) guide. To enable offline support, please follow the given steps:
```

this should be changed to v5 to v6. For later PR i think. But mentioning here to not forget.

Reply:
yes